vnext/Shared/Threading/MessageDispatchQueue.h

```diff
@@ -43,4 +44,34 @@ struct MessageDispatchQueue : facebook::react::MessageQueueThread, std::enable_s
   const Mso::Promise<void> m_whenQuit;
 };
 
+struct MessageSimpleDispatchQueue : facebook::react::MessageQueueThread,
```

Comment:
>MessageSimpleDispatchQueue [](http://example.com/codeflow?start=7&length=26)

Duplicating of another class is a good sign that we must chnage the IDispatchQueue interface instead. :)

Reply:
Filed #8602 to followup.